src/lavatory/commands/stats.py

```diff
@@ -0,0 +1,24 @@
+"""Statistics of the repo."""
+import logging
+
+import click
+
+from ..utils.artifactory import Artifactory
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command()
+@click.pass_context
+@click.option(
+    '--repo',
```

Comment:
This flag should match the `--repo` flag in `purge`. in `purge` you can specify `--repo` multiple times and I think should should behave the same for consistency.

Reply:
I was wondering, how that would work multiple?

Reply:
The same as it does for "all repos" just with a limited list. If you add `multiple=True,` to the Click command, it will pass a list of all the values, like `--repo test --repo other -- repo more` would pass in `["test", "other", "more"]`.

Reply:
When you give multiple did it work? Yeah easily I can make it work if it comes down as a list.

Reply:
Yeah, it works great in the `purge` command. You can see how it was implemented at https://github.com/gogoair/lavatory/blob/master/src/lavatory/commands/purge.py#L29 and here is an example of the output where I specify two repo:

```
$ lavatory purge --repo docker-local --repo vagrant-iteration
[INFO] lavatory.utils.setup_pluginbase Searching for policies in ['/Users/dcampbell/work/lavatory/src/lavatory/utils/../policies']
[INFO] lavatory.commands.purge Applying retention policies to docker-local, vagrant-iteration
....
```

Reply:
I have made it, it does not come as list but tuple. Corrected according to that.